packages/api-schemas/src/notifications.schemas.ts

```diff
@@ -0,0 +1,355 @@
+                operator: z.literal("or"),
+                value: z
+                  .array(
+                    z
+                      .object({
+                        operator: z.union([z.literal("eq"), z.literal("lt"), z.literal("gt"), z.literal("lte"), z.literal("gte")]),
+                        field: z.string(),
+                        value: z.union([z.string(), z.number(), z.boolean()])
+                      })
+                      .passthrough()
+                  )
+                  .min(2)
+              })
+              .passthrough(),
+            z
+              .object({
+                operator: z.union([z.literal("eq"), z.literal("lt"), z.literal("gt"), z.literal("lte"), z.literal("gte")]),
+                field: z.string(),
+                value: z.union([z.string(), z.number(), z.boolean()])
+              })
+              .passthrough()
+          ])
+        })
+        .passthrough(),
+      z
+        .object({
+          userId: z.string().uuid(),
+          contactPointId: z.string().uuid(),
+          enabled: z.boolean(),
+          summary: z.string().min(3),
+          description: z.string().min(3),
+          id: z.string().uuid(),
+          status: z.string(),
+          createdAt: z.unknown(),
+          updatedAt: z.unknown(),
+          type: z.literal("DEPLOYMENT_BALANCE"),
+          conditions: z.union([
+            z
+              .object({
+                operator: z.literal("and"),
+                value: z
+                  .array(
+                    z
+                      .object({
+                        operator: z.union([z.literal("eq"), z.literal("lt"), z.literal("gt"), z.literal("lte"), z.literal("gte")]),
+                        field: z.literal("balance"),
+                        value: z.number()
+                      })
+                      .passthrough()
+                  )
+                  .min(2)
+              })
+              .passthrough(),
+            z
+              .object({
+                operator: z.literal("or"),
+                value: z
+                  .array(
+                    z
+                      .object({
+                        operator: z.union([z.literal("eq"), z.literal("lt"), z.literal("gt"), z.literal("lte"), z.literal("gte")]),
+                        field: z.literal("balance"),
+                        value: z.number()
+                      })
+                      .passthrough()
+                  )
+                  .min(2)
+              })
+              .passthrough(),
+            z
+              .object({
+                operator: z.union([z.literal("eq"), z.literal("lt"), z.literal("gt"), z.literal("lte"), z.literal("gte")]),
+                field: z.literal("balance"),
+                value: z.number()
+              })
+              .passthrough()
+          ]),
+          params: z.object({ dseq: z.string(), owner: z.string() }).passthrough()
+        })
+        .passthrough()
```

Comment:
I was looking in `passthrough` and it redirects to `looseObject` https://zod.dev/api?id=zlooseobject. Do we want to have the objects loose and accept unknown keys?

Reply:
wait, I'm dropping these generated schemas, there is no need for them at all for now